lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -15,6 +15,7 @@ module ExternalHttpdConfiguration
       SSSD_CONFIG          = "/etc/sssd/sssd.conf".freeze
       PAM_CONFIG           = "/etc/pam.d/httpd-auth".freeze
       HTTP_KEYTAB          = "/etc/http.keytab".freeze
+      HTTP_CONFIG_FILE     = "/etc/httpd/conf.d/manageiq-redirects-ws".freeze
```

Comment:
right, I don't think we document that evmserverd must be running for configuring external auth, thus the above check which essentially says appliance was previously started and all these httpd files were installed earlier.  Otherwise a simple check that both evmserverd & httpd are running would suffice.

@jvlcek one other possibility is restart httpd only if it is currently running (i.e. evmserverd is up and running and we want the changes made in effect immediately).